Document default Electron offline support (#5758)
getsentry/sentry-docs: src/platforms/javascript/guides/electron/index.mdx

```

## Offline Support

> The default transport automatically handles offline caching of events. It has a
> number of options that allow customizing queueing behavior:

Reviewer: ```suggestion
The default transport automatically handles offline events caching. Following, are a
number of options that allow you to customize queueing behavior:
```
